tests/lib/external/snapd-testing-tools/tests/os.query/task.yaml

```diff
@@ -163,7 +163,6 @@ execute: |
         opensuse-15.5-64)
             os.query is-opensuse
             os.query is-opensuse 15.5
-            ! os.query is-opensuse tumbleweed
```

Comment:
Is this a drive-by fix? Seems unrelated to lunar

Reply:
yes, but I merged the changes in snapd-testing-tools project and it was already merged in main branch